### ISSUE TITLE
Lower jig v3.2 Improvements

### ISSUE THREAD

**Issue by swolebro** [swolebro/swolebro-youtube#3]
Some notes to myself on things to improve on the next version, based on my use of v3.1:

* replace the thread notch with a simple "v" notch at each end for aligning the initial scribing template (via precision eyeballing)
* change the spacing/sizing on the pocket pre-drill template to be less ambitious
* either develop a second set of side plates for full-size holes, or maybe some sort of bushing system for them
* give the milling template a flat sidewall section prior to starting the loft (would make alignment by feel easier)
* make the amount of offset between the blueprint pocket dimensions and the template pocket configurable (currently 0.000", but 0.003" would probably be good)
* adjust the offset of the trigger slot template to match the above
* some offset for the trigger slot template from the bottom of the pocket
* a relief on the slide plate trigger area for clearing when milling out the trigger slot

Maybe's:
* printed buffer tube adapter
* maybe slots for drop-in nuts, rather than threading the printed parts

**Comment by swolebro** [comment]
The updates are done and I've recorded a new video for it too. Now in the process of editing it (fun!). The goal is to make this one a quicker run through, since the last one was stupidly long.

If you want to get a headstart, I've chucked the latest files up on Thingiverse as v3.2beta17. That includes a few cosmetic tweaks done as a result of the video (basically, adding chamfers), and I'm pretty sure that's a wrap. Very pleased with how it works now. There **shouldn't** be any need for a v3.3.

Once the video is edited and published, I'll put the CAD files up on LBRY too. The only reason for the hold-up is I want to put the video link in the filepack README. Hahah.
